examples/opmi.cpp

```diff
@@ -28,6 +28,15 @@
 #include <opm/parser/eclipse/EclipseState/SummaryConfig/SummaryConfig.hpp>
 #include <opm/parser/eclipse/EclipseState/Schedule/Schedule.hpp>
 
+#include <opm/common/OpmLog/OpmLog.hpp>
+#include <opm/common/OpmLog/StreamLog.hpp>
+#include <opm/common/OpmLog/LogUtil.hpp>
+
+
+void initLogging() {
```

Comment:
You could have called this function:

```
    void OpmLog::setupSimpleDefaultLogging(const bool use_prefix,
                                           const bool use_color_coding,
                                           const int message_limit)
```
with `(false, false, 0)` parameters to get the same, on the other hand maybe you want to customize the logging here at some point?

Reply:
OK - that is what I wanted - will update.